phoenix-core/src/test/java/org/apache/phoenix/index/IndexUpgradeToolTest.java

```diff
@@ -125,6 +125,30 @@ public void testIfOptionsArePassedToIndexTool() {
         it.populateIndexToolAttributes(commandLine);
     }
 
+    @Test
```

Comment:
If you prefer to have a new test for this, please consider removing duplicate code in a function that can be called from both tests.

Reply:
The test is to ensure that we can override the properties through IndexUpgradeTool too and that the properties are passed correctly to IndexTool and can be parsed there. I've taken out the index tool parsing part which was common with already written tests.